fedora/jellyfin-selinux-launcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /usr/lib64/jellyfin/jellyfin ${@}
```

Comment:
You should [quote the `$@` to prevent expansion problems](https://www.shellcheck.net/wiki/SC2068) with arguments that might have spaces in them.

```suggestion
exec /usr/lib64/jellyfin/jellyfin "${@}"
```